AUTHORS

```diff
@@ -0,0 +1,6 @@
+Antonio Ojea <6450081+aojea@users.noreply.github.com>
```

Comment:
Why this github email?

Reply:
Its probably automatically generated by PBR using the authors email. I believe those addresses are generated by github when responding to issues via email so not sure where its coming from??

Reply:
It is automatically populated from git history:

commit 667a9484b1b148b889fe8708cb140553feff135c
Merge: c48755e 7937e91
Author: Antonio Ojea <6450081+aojea@users.noreply.github.com>

Reply:
can we add my corporate email instead? aojeagarcia@suse.com looks better

Reply:
They are both included. I can investigate how to filter that one commit out of pbr's world view.